.github/workflows/firebaseai.yml

```diff
@@ -162,3 +163,14 @@ jobs:
       run: sed -i "" "s#s.swift_version = '5.9'#s.swift_version = '${{ matrix.swift_version}}'#" FirebaseAI.podspec
     - name: Build and test
       run: scripts/third_party/travis/retry.sh scripts/pod_lib_lint.rb FirebaseAI.podspec --platforms=${{ matrix.target }} ${{ matrix.warnings }}
+
+  quickstart:
+    # Verifies the quickstart builds with this PR. Only run on pulls where branch is available.
+    if: github.event_name == 'pull_request'
```

Comment:
No need to address this now but I think the branch name may be `main` if we wanted to run this on the nightly schedule as well. Untested but I saw `GITHUB_REF` "Default branch" in https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#schedule.